specification/2.0/README.md

```diff
@@ -191,6 +191,10 @@ glTF uses URIs to reference buffers and image resources. Clients must support at
 
  > **Implementation Note:** This allows the application to decide the best approach for delivery: if different assets share many of the same geometries, animations, or textures, separate files may be preferred to reduce the total amount of data requested. With separate files, applications can progressively load data and do not need to load data for parts of a model that are not visible. If an application cares more about single-file deployment, embedding data may be preferred even though it increases the overall size due to base64 encoding and does not support progressive or on-demand loading. Alternatively, an asset could use GLB container to store JSON and binary data in one file without base64 encoding. See [GLB File Format Specification](#glb-file-format-specification) for details.
 
+Applications should consider applying syntax-based normalization to URIs as defined by [RFC&nbsp;3986, Section&nbsp;6.2.2.](https://tools.ietf.org/html/rfc3986#section-6.2.2), [RFC&nbsp;3987, Section&nbsp;5.3.2.](https://tools.ietf.org/html/rfc3987#section-5.3.2), and applicable schema rules (e.g., [RFC&nbsp;7230, Section&nbsp;2.7.3.](https://tools.ietf.org/html/rfc7230#section-2.7.3) for HTTP) on export and/or import.
+
+> **Implementation Note:** While the spec does not explicitly disallow non-normalized URIs, their usage may lead to unwanted side-effects such as security warnings or cache misses on some platforms.
```

Comment:
Is it worth mentioning that some client implementations may reject non-normalized URIs for security reasons? E.g. I would expect Facebook to do so. The side effects could potentially be a little more severe than warnings or cache misses.

Reply:
Valid point re per-app rejection. Could you please adjust the note?

Reply:
Done